test_infra/stacks/base_stack.py

```diff
@@ -20,6 +20,14 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs: str) -> None:
             enable_dns_support=True,
         )
         Tags.of(self.vpc).add("Name", "aws-sdk-pandas")
+
+        # Add outputs for the EIPs generated by the VPC construct
+        for public_subnet in self.vpc.public_subnets:
+            elastic_ip: ec2.CfnEIP = public_subnet.node.find_child("EIP")  # type:ignore
+            CfnOutput(
+                self, f"{public_subnet.node.id} EIP", value=elastic_ip.ref, export_name=f"{public_subnet.node.id}EIP"
```

Comment:
Just noticed `export_name=f"{public_subnet.node.id}EIP"` would this not create unique exports that would cause having to update all dependent stacks if they change? Can we avoid relying on stack exports here and use SSM, for example?